torchaudio/csrc/transducer.cpp

```diff
@@ -0,0 +1,82 @@
+#ifdef BUILD_TRANSDUCER
```

Comment:
Instead of guarding the whole file, we should exclude this from the list of source code when not building transducer binding.

Reply:
addressed in #1159